plugin/commentary.vim

```diff
@@ -87,6 +126,12 @@ if !hasmapto('<Plug>Commentary') || maparg('gc','n') ==# ''
   nmap gcc <Plug>CommentaryLine
   nmap cgc <Plug>ChangeCommentary
   nmap gcu <Plug>Commentary<Plug>Commentary
+  xmap gcy   <Plug>CommentaryYank
+  nmap gcy   <Plug>CommentaryYank
+  nmap gcyy  <Plug>CommentaryYankLine
+  xmap gcd   <Plug>CommentaryDupe
+  nmap gcd   <Plug>CommentaryDupe
```

Comment:
I am indeed adopting the parentheses convention for new plugins, but I will not be updating old plugins unless there's a concrete issue.